web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/forvaltning/StønadskontoMigreringURTask.java

```diff
@@ -0,0 +1,127 @@
+import no.nav.foreldrepenger.domene.json.StandardJsonConfig;
+import no.nav.foreldrepenger.domene.uttak.beregnkontoer.BeregnStønadskontoerTjeneste;
+import no.nav.foreldrepenger.stønadskonto.grensesnitt.Stønadsdager;
+import no.nav.foreldrepenger.stønadskonto.regelmodell.grunnlag.LegacyGrunnlagV0;
+import no.nav.foreldrepenger.stønadskonto.regelmodell.grunnlag.LegacyGrunnlagV1;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+@Dependent
+@ProsessTask(value = "stønadskonto.migrering", maxFailedRuns = 1)
+@FagsakProsesstaskRekkefølge(gruppeSekvens = false)
+class StønadskontoMigreringURTask implements ProsessTaskHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(StønadskontoMigreringURTask.class);
+    private static final String FRA_ID = "fraId";
+    private static final String DRY_RUN = "dryRun";
+
+    private final FagsakRelasjonRepository fagsakRelasjonRepository;
+    private final EntityManager entityManager;
+    private final ProsessTaskTjeneste prosessTaskTjeneste;
+    private final UttakInputTjeneste uttakInputTjeneste;
+    private final FagsakRelasjonRepository fagsakRelasjonTjeneste;
+    private final BeregnStønadskontoerTjeneste beregnStønadskontoerTjeneste;
+
+    @Inject
+    public StønadskontoMigreringURTask(FagsakRelasjonRepository fagsakRelasjonRepository,
+                                       EntityManager entityManager,
+                                       ProsessTaskTjeneste prosessTaskTjeneste,
+                                       UttakInputTjeneste uttakInputTjeneste,
+                                       FagsakRelasjonRepository fagsakRelasjonTjeneste,
+                                       BeregnStønadskontoerTjeneste beregnStønadskontoerTjeneste) {
+        this.fagsakRelasjonRepository = fagsakRelasjonRepository;
+        this.entityManager = entityManager;
+        this.prosessTaskTjeneste = prosessTaskTjeneste;
+        this.uttakInputTjeneste = uttakInputTjeneste;
+        this.fagsakRelasjonTjeneste = fagsakRelasjonTjeneste;
+        this.beregnStønadskontoerTjeneste = beregnStønadskontoerTjeneste;
+    }
+
+    @Override
+    public void doTask(ProsessTaskData prosessTaskData) {
+        var fraId = Optional.ofNullable(prosessTaskData.getPropertyValue(FRA_ID)).map(Long::valueOf).orElse(null);
+        var dryRun = Optional.ofNullable(prosessTaskData.getPropertyValue(DRY_RUN)).filter("false"::equalsIgnoreCase).isEmpty();
+
+        var beregninger = finnNesteHundreStønadskonti(fraId).toList();
+
+        beregninger.forEach(ur -> håndterBeregning(ur, dryRun));
+
+        beregninger.stream()
```

Comment:
Er vi sikker på at vi ønsker å ta alt i en sleng fra en gitt id? Tipper det er større sannsynlighet for at noe feiler av de tidligeste jobbene.